asset-managment/niapm.yml

```diff
@@ -0,0 +1,1405 @@
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
+    post:
+      tags: [assets]
+      summary: Creates a new calibration history entry for an asset
+      description: Creates a new calibration history entry for a non-NI manufactured asset.
+      operationId: post-asset-calibration-history
+      x-ni-operation: writeAssets
+      x-ni-priviledge: Write
+      parameters:
+        - $ref: '#/parameters/AssetId'
+        - $ref: '#/parameters/ExternalCalibrationRequestBody'
+      responses:
+        201:
+          description: Created
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
+  /v1/assets/{assetId}/history/delete-calibrations:
+    post:
+      tags: [assets]
+      summary: Deletes calibration history entries
+      description: Deletes multiple calibration history entries for an asset.
+      operationId: post-delete-asset-calibration-history
+      x-ni-operation: deleteAssets
+      x-ni-priviledge: Write
+      parameters:
+        - $ref: '#/parameters/AssetId'
+        - $ref: '#/parameters/DeleteCalibrationsRequestBody'
+      responses:
+        200:
+          $ref: '#/responses/PartialSuccess'
+        204:
+          description: No Content
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
+  /v1/assets/{assetId}/history/query-calibration:
+    post:
+      tags: [assets]
+      summary: Query asset calibration history
+      description: Computes the calibration history of an asset in CSV format. If the destination is download, the response will contain the calibration history file content. If the destination is file ingestion, the CSV file is sent to the file storing service and an object containing the file identifier is returned. Any other format and destination combinations are not supported yet.
+      operationId: post-asset-query-calibration-history
+      x-ni-operation: readAssets
+      x-ni-priviledge: Read
+      parameters:
+        - $ref: '#/parameters/AssetId'
+        - $ref: '#/parameters/QueryHistoryRequestBody'
+      responses:
+        200:
+          $ref: '#/responses/SendToFileIngestionOrDownloadResponse'
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+        500:
+          description: Internal server error. Possible while sending the file to the FileIngestion service, because the service is not running or there is not enough space on disk to store the file or the file could not be sent due to networking issues.
```

Comment:
"Possible while sending the file to the FileIngestion service because the service is not running; there is not enough space on disk to store the file; or there are network issues."?